### PR TITLE
updated split_search

### PR DESCRIPTION
**Pull request by tkosciol**
solving issues #33 #34 and #35

### CODE REVIEW

File: microprot/scripts/split_search.py [hunk 1]
```diff
@@ -509,27 +520,27 @@ def mask_sequence(hhsuite_fp, fullsequence_fp, subsequences_fp=None,
 
     # read the original protein file, used to run HHsearch
     p = Protein.read(fullsequence_fp, seq_num=1)
-    queryname = p.metadata['id'] + ' ' + p.metadata['description']
+    queryname = [p.metadata['id'], p.metadata['description']]
```
Comment (sjanssen2) [inline microprot/scripts/split_search.py]: maybe we should have more speaking variable names for those two fields of the queryname?

**Comment by tkosciol** [comment]
should I just remove `pass` from `pretty_output` instead? I thought it's customary in Python to have some sort of return value in each function.

**Comment by sjanssen2** [comment]
I don't know the best pythonic practice, but I find it confusing. If there is no return value then just remove the "pass", please.

**Comment by sjanssen2** [comment]
Is it too much to ask you for adding more unit-tests to cover those 11 missed lines of code?

**Comment by tkosciol** [comment]
I ain't doing tests for click, because it's difficult and it doesn't make much sense, i.e. click uses `mask_sequence` function and `pretty_output`, both of which are tested.
Are you cool with that?

**Pull request merged**